Lib/test/test_urlparse.py

```diff
@@ -566,6 +566,21 @@ def test_urlsplit_attributes(self):
         self.assertEqual(p.port, 80)
         self.assertEqual(p.geturl(), url)
 
+        # Remove ASCII tabs and newlines from input
```

Comment:
super long test methods get annoying, how about creating a new test method for these added test and assertion calls.